### DISCUSSION TITLE
How to hide artboard names on the canvas?

### DISCUSSION THREAD

**Discussion by mscofield0** [uxbox/uxbox#1205] [Q&A]
The artboard names just cover up the whole screen. How do I hide them?

**Comment by elhombretecla** [comment]
Hi @mscofield0 thanks for your questions :)

This option is not available right now. We are going to manage this as an enhancement if you are agree :)

**Reply by mscofield0** [reply]
I agree! Thank you.

**Reply by elhombretecla** [reply]
Great!!! We just created an enhancement for that :)

You can track it here [https://tree.taiga.io/project/penpot/issue/2034](https://tree.taiga.io/project/penpot/issue/2034)
